### PR TITLE
[DOCS] installation guide restructuring 23.1

### PR DESCRIPTION
**Pull request by kblaszczak-intel**

### CODE REVIEW

File: docs/install_guides/installing-openvino-vcpkg.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# Install OpenVINO™ Runtime via Homebrew {#openvino_docs_install_guides_installing_openvino_vcpkg}
```
Comment (artanokhov) [inline docs/install_guides/installing-openvino-vcpkg.md:1]: Need to updated with 'Install OpenVINO™ Runtime via VCPKG'

File: docs/install_guides/installing-openvino-overview.md [hunk 2]
```diff
@@ -25,25 +23,43 @@
...
+   All currently supported versions are:
+
+   * 2023.1 (development)
+   * 2022.3 (LTS)
+   * 2021.4 (LTS) 
+
+   Moreover, different OpenVINO distributions may support slightly different sets of features.
+   Read installation guides for particular distributions for more details. 
+
+   .. dropdown:: Distribution Comparison
```
Comment (artanokhov) [inline docs/install_guides/installing-openvino-overview.md:48]: Do we want to mention that this table applicable for 2023.1?

**Review by artanokhov — CHANGES_REQUESTED** [review]

File: docs/install_guides/installing-openvino-windows-header.md [hunk 3]
```diff
@@ -15,7 +15,9 @@
    Use Archive <openvino_docs_install_guides_installing_openvino_from_archive_windows>
    Use PyPI <openvino_docs_install_guides_installing_openvino_pip>
    Use Conda Forge <openvino_docs_install_guides_installing_openvino_conda>
-   Use Docker <openvino_docs_install_guides_installing_openvino_docker>
+   Use VCPKG <openvino_docs_install_guides_installing_openvino_vcpkg>
+   Use Docker <openvino_docs_install_guides_installing_openvino_docker_linux>
```
Comment (kblaszczak-intel) [inline docs/install_guides/installing-openvino-windows-header.md:19]:
```suggestion
   Use Docker <openvino_docs_install_guides_installing_openvino_docker>
```

**Review by sgolebiewski-intel — APPROVED** [review]

File: docs/install_guides/installing-openvino-apt.md [hunk 4]
```diff
@@ -1,36 +1,38 @@
...
+   * additionally includes code samples 
+   * is dedicated to Linux users (may offer different hardware support under different 
+     operating systems, see the drop-down below for more details).
 
-   By downloading and using this container and the included software, you agree to the terms and 
-   conditions of the 
-   `software license agreements <https://software.intel.com/content/dam/develop/external/us/en/documents/intel-openvino-license-agreements.pdf>`__.
+   .. dropdown:: Inference Options 
 
+      ===================  =====  =====  =====  =====  ========  =============  ========  ========
```
Comment (artanokhov) [inline docs/install_guides/installing-openvino-apt.md:21]: For APT we activating only RPMs/DEBs from following OSes:
Ubuntu18, Ubuntu20, Ubuntu22, RHEL8

File: docs/install_guides/installing-openvino-conda.md [hunk 5]
```diff
@@ -4,14 +4,35 @@
...
+
+   * offers both C++ and Python APIs
+   * does not offer support for GNA and NPU inference
+   * is dedicated to users of all major OSs: Windows, Linux, macOS.
+   * may offer different hardware support under different operating systems
+     (see the drop-down below for more details)
+
+   .. dropdown:: Inference Options 
+
+      ===================  =====  =====  =====  =====  ========  =============  ========  ========
```
Comment (artanokhov) [inline docs/install_guides/installing-openvino-conda.md:22]: NPU not presented for Conda-Forge at all, as well as GNA

File: docs/install_guides/installing-openvino-pip.md [hunk 6]
```diff
@@ -7,27 +7,63 @@
...
+   
+   * offers the Python API only
+   * does not offer support for GNA and NPU inference
+   * is dedicated to users of all major OSs: Windows, Linux, macOS
+   * may offer different hardware support under different operating systems
+     (see the drop-down below for more details)
+
+   .. dropdown:: Inference Options 
+
+      The PyPi distribution for macOS currently supports the arm64 architecture only.
```
Comment (artanokhov) [inline docs/install_guides/installing-openvino-pip.md:22]: We activate x86_64 as well - as for example
https://files.pythonhosted.org/packages/4d/70/ac75b270b7e33a28aab12b9dec2a47e63202457285bc35c1b3454cb43229/openvino-2023.0.2-11065-cp311-cp311-macosx_10_12_x86_64.whl

File: docs/install_guides/installing-openvino-pip.md [hunk 7]
```diff
@@ -7,27 +7,63 @@
...
+      
+      ===================  =====  =====  =====  =====  ========  =============  ========  ========
```
Comment (artanokhov) [inline docs/install_guides/installing-openvino-pip.md:24]: NPU and GNA are not presented for PyPI distr. channel.

**Pull request merged**